### DISCUSSION TITLE
As of few days ago I suddenly see all ads

### DISCUSSION THREAD

**Discussion by NewbieCPlusPlus** [uazo/bromite-buildtools#251] [General]
The ads within the browser were non-existent across all websites, but a few days ago ads started leaking on all websites I regularly visit.

I noticed ads started leaking in version v114.0.5735.110, so I updated to the latest version v114.0.5735.134, but the ads are still present. 

I haven't touched security settings and your blocking list is still enabled in the browser's settings, just like it always was.

Is there a particular reason why the browser stopped blocking ads?

**Comment by uazo** [comment]
eyeo adblock also reads from the metadata of the downloaded rules a possible value related to the "validity duration" of the rules (https://eyeo.gitlab.io/adblockplus/abc/core-spec/#appendix-filter-list-syntax), which by default is 5 days (`kDefaultExpirationInterval`).
it probably happened that the rules expired but were not re-downloaded in the meantime: try deselecting and reselecting the rules.

it becomes more and more urgent to review the functions of adblock a bit, especially alerts towards the user
thank you.

**Reply by NewbieCPlusPlus** [reply]
Deselecting and reselecting rules worked!

Thanks
